src/app/components/toolbar/toolbar-guidelines.component.html

```diff
@@ -48,13 +42,12 @@
 </section>
```

Comment:
I know the text in the Tile toolbar section was already in place on the toolbar component docs, but reading through it again, "It contains actions to manipulate the contents and display of the tile or a portion of the tile" reads awkwardly. At least the "display of the tile" part does. I'd change "display" to "appearance" unless we mean something else by "display" here.

Reply:
see above